lib/index.ts

```diff
@@ -110,22 +111,13 @@ export async function inspect(
     );
     const parseResult = parseTree(result, options.dev);
     const { javaVersion, mavenVersion } = parseVersions(versionResult);
-    let callGraph: CallGraph | undefined;
+    let callGraph: CallGraphResult | undefined;
     let maybeCallGraphMetrics = {};
     if (options.reachableVulns) {
-      debug(`getting call graph from path ${targetPath}`);
-      try {
-        callGraph = await javaCallGraphBuilder.getCallGraphMvn(
-          path.dirname(targetPath),
-        );
-        maybeCallGraphMetrics = {
-          callGraphMetrics: javaCallGraphBuilder.runtimeMetrics(),
-        };
-        debug('got call graph successfully');
-      } catch (err) {
-        debug('call graph error: ' + err);
-        throw new CallGraphError(err.message, err);
-      }
+      callGraph = await getCallGraph(targetPath, options?.callGraphBuilderTimeout);
+      maybeCallGraphMetrics = {
+        callGraphMetrics: javaCallGraphBuilder.runtimeMetrics(),
```

Comment:
Also will be nice to add the timeout to the metrics, this we will be able to see it in the logs